lazycatlabs/flutter_auth_app
# feat: improve some code implementation

## What it does

- enable validation on user-interaction
- reduce setState usage
- resolve the test
- resolve the Github action error when creating PR to the main branch
- use dart-define to define env data
- update the docs

## How to test

1. Install the app
2. Test to fill the text field
3. Observe the view
4. Run `flutter test -j1` to run the test

## Screenshot
n/a
